nats/nats.go

```diff
@@ -36,14 +36,14 @@ func init() {
 
 // logErrorw is a helper function to log error messages
 func (s *Storage) logErrorw(msg string, keysAndValues ...interface{}) {
-	if s.cfg.UseLogger && s.cfg.Logger != nil {
+	if s.cfg.Logger != nil {
```

Comment:
I actuallt think we need the check 😂 but just the first one. This since the var will have a default value of either True or False. It won't be nil at any point

Reply:
so `if s.cfg.UseLogger { ... }` should be enough

Reply:
@cr1cr1 Does the above make sense?

Reply:
Indeed, Logger is never nil when using the bool option, renamed to Verbose